### PR TITLE
fix: presented credentials should not be stored

### PR DESCRIPTION
**Pull request by artursudnik**
This PR removes logic that persists presented credentials

### CODE REVIEW

**Review by jrhender — COMMENTED** [review]
Looks like a good start. I think some additional changes might be needs to ensure that the submitted VP is still returned in the callback https://github.com/energywebfoundation/ssi/blob/e402c40992d6cfabcbfccfd03fbe95e5c625b48c/apps/vc-api/src/vc-api/exchanges/exchange.service.ts#L118 
We might need a test that checks that as well as the test don't seem to have failed 😅 . I think we're checking that there is a callback but maybe not what the data is in the callback https://github.com/energywebfoundation/ssi/blob/e402c40992d6cfabcbfccfd03fbe95e5c625b48c/apps/vc-api/test/vc-api/exchanges/resident-card/resident-card.e2e-suite.ts#L106

**Comment by artursudnik** [comment]
Good point. I'll try to add tests to prevent any regressions here as this is a risky change.

**Comment by artursudnik** [comment]
I hope that when we have good unit tests, it is not necessary to check in e2e tests if a callback endpoint has been called correctly

**Comment by jrhender** [comment]
Yes, fair, it might be better to have unit tests that cover this.

File: apps/vc-api/src/vc-api/exchanges/dtos/presentation-submission-full.dto.ts [hunk 1]
```diff
@@ -16,24 +16,26 @@
...
 
 import { VerificationResultDto } from '../../credentials/dtos/verification-result.dto';
-import { ValidateNested } from 'class-validator';
+import { IsNotEmpty, ValidateNested } from 'class-validator';
 import { VerifiablePresentationDto } from '../../credentials/dtos/verifiable-presentation.dto';
 import { Type } from 'class-transformer';
 
 /**
  * Presentation Submission Dto.
  */
```
Comment (jrhender) [inline apps/vc-api/src/vc-api/exchanges/dtos/presentation-submission-full.dto.ts]:
```suggestion
/**
 * Presentation Submission Full Dto.
 * The complete presentation submission, including the submitted Verifiable Presentation
 */
```

File: apps/vc-api/src/vc-api/exchanges/dtos/presentation-submission-secure.dto.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { VerificationResultDto } from '../../credentials/dtos/verification-result.dto';
+import { ValidateNested } from 'class-validator';
+import { VerifiablePresentationDto } from '../../credentials/dtos/verifiable-presentation.dto';
+import { Type } from 'class-transformer';
+
+/**
+ * Presentation Submission Dto.
```
Comment (jrhender) [inline apps/vc-api/src/vc-api/exchanges/dtos/presentation-submission-secure.dto.ts]:
```suggestion
 * Secure Presentation Submission Dto
 * A representation of a presentation submission which does not have personal data associated with the submission
```

**Review by jrhender — APPROVED** [review]
Looks good. I added a couple documentation suggestions but I think it looks good as is

**Pull request merged**